lte/gateway/c/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -222,6 +228,7 @@ void MmeNasStateManager::free_state() {
     return;
   }
   clear_mme_nas_hashtables();
+  clear_mme_ueip_imsi_map();
```

Comment:
when is `free_state` executed on mme? I mean, in what situation de we need to clear all those tables?

Reply:
On mme exit or on mme restart,  mme clears all the in-memory contexts. free_state() is invoked to clear in memory contexts